UT4MasterServer.Web/src/services/authentication.service.ts

```diff
@@ -81,6 +82,7 @@ export default class AuthenticationService extends HttpService {
             console.error('Error killing session', err);
         }
         finally {
+            AccountStore.account = null;
```

Comment:
This is the bug fix, needed to clear the account on log out, otherwise it doesn't fetch it again and retains the account of the first user to log in

Reply:
Can you write a helper function with `AccountStore.Clear()` and `SessionStore.Clear()` to have it in one place and able to use it in for example when returning `UNAUTHORIZED` error from backend?

Reply:
That would be this function. This is the one place.

Reply:
I thought about having a clear function on the stores, but each only has the one prop to clear, so I left it out for now. As for the single place to call in order to end the session and log the user out, that would be this function. Having clear functions on the stores wouldn't be one place to accomplish this, it would be two.

Reply:
As for logging out when receiving a 401 from an API request, I'm not sure that is always the thing to do. There may be some calls that it would make sense for, but I could see cases where a 401 response would just be the normal way to tell a user they weren't allowed to do something and it shouldn't log them out then. I decided to just not do any automatic logging out based on 401s for now, that could change at some point.